src/Question07.java

```diff
@@ -0,0 +1,56 @@
+
+import java.util.Scanner;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author Sami
+ */
+public class Question07 {
+
+    public static void main(String[] args) {
+
+        //make a scanner
+        Scanner input = new Scanner(System.in);
+
+        //user enters speed limit
+        System.out.print("Enter the speed limit: ");
+        int speedLimit = input.nextInt();
+
+        //user enters speed of car
+        System.out.print("Enter the recorded speed of the car: ");
+        int carSpeed = input.nextInt();
+
+        //three fines
+        int firstFine = 100;
+        int secondFine = 270;
+        int thirdFine = 500;
+
+        //checks how much they are speeding by
+        int speedOver = carSpeed - speedLimit;
+
+//I added another step just for fun

```

Comment:
I added another part to question 7. its just a part that says to speed up if they are under the speed limit by 15km/h or more